tests/10apidoc/12device_management.pl

```diff
@@ -10,6 +10,19 @@ sub matrix_get_device {
    );
 }
 
+sub matrix_set_device_display_name {
```

Comment:
Hrm. This brace should have been on its own line, but then so should the other functions in the file. Don't worry about it; I'll do a style fixup later.